app/controllers/users_controller.rb

```diff
@@ -1,3 +1,25 @@
 class UsersController < ApplicationController
-  def new; end
+  def new
+    @user = User.new
+  end
+
+  def show
+    @user = User.find(params[:id])
+  end
+
+  def create
+    @user = User.new(user_params)
```

Comment:
```suggestion
    @user = User.new user_params
```
hạn chế dùng ngoặc nhé e